src/molecules/BreadcrumbBar.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
```

Comment:
This file is not used in the story. We _can_ delete it.

If we want to keep it - and I think that maybe we should? – maybe something like ...
```jsx
import { HorizontalLinkBar } from './HorizontalLinkBar'
import { BreadcrumbItem } from './BreadcrumbItem'

const BreadcrumbBar = props => <HorizontalLinkBar ListItemElement={BreadcrumbItem} {...props} />

export { BreadcrumbBar }
```
... will be enough?

And use it in the story as well.